readme.md

```diff
@@ -1,14 +1,19 @@
 # lc-validator-fields-match
 
 This is a [lc-form-validation](https://github.com/Lemoncode/lcFormValidation) add-on that brings validation capabilities for:
-// TODO:
+* Validate if two fields match.
 
 Please, refer to [lc-form-validation](https://github.com/Lemoncode/lcFormValidation) to know more.
 
```

Comment:
Let's complete this part (we will do the same in the DNI), keep the parapgrah indicating that is lc-form-validation...

First we have to add a more comprehensive explanation, for instance:

```
Validate if two fields match: check if two fields have the same value, sample usage: validate if password and repeat password fields match, validate if email and repeat email fields match...
```

We should add a link to codesandbox or stackblitz with sample running (first we have to publish it on github)

Add some code indicating a sample usage

Reply:
When @Nasdan publishes it, I'll add the samples.